packages/terra-heading/src/Heading.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import 'terra-base/lib/baseStyles';
+import './Heading.scss';
+
+const propTypes = {
+  /**
+   * Needs descriptions
+   */
+  children: PropTypes.node.isRequired,
+  /**
+   * The color of the text. Accepts a CSS color value.
+   */
+  color: PropTypes.string,
+  /**
+   * Sets the text to display in italics.
+   */
+  isItalic: PropTypes.bool,
+  /**
+   * Sets the text to appear visually hidden. The text will still be available to screen readers.
+   */
+  isVisuallyHidden: PropTypes.bool,
+  /**
+   * Sets the heading level. One of 1, 2, 3, 4, 5, 6.
+   */
+  level: PropTypes.oneOf([1, 2, 3, 4, 5, 6]).isRequired,
+  /**
+   * Sets the text size. One of mini, tiny, small, medium, large, huge.
+   */
+  size: PropTypes.oneOf(['mini', 'tiny', 'small', 'medium', 'large', 'huge']),
+  /**
+   * Sets the text size. One of 200, 400, 700.
+   */
+  weight: PropTypes.oneOf([200, 400, 700]),
+};
+
+const defaultProps = {
```

Comment:
Updated: https://github.com/cerner/terra-core/pull/470/commits/c6a552c630b6f2d8658c50c2ea94e8b43bfe4a64

I set the default prop for weight to 700. I'd like to keep the level/size un-defaulted so consumers spend time thinking about there [heading levels / nesting structure](https://www.w3.org/WAI/tutorials/page-structure/headings/#heading-ranks).